monitor.go

```diff
@@ -172,8 +172,8 @@ func (m *AliveMessage) Header() http.Header {
 // MaxAge extracts "max-age" value from "CACHE-CONTROL" property.
 func (m *AliveMessage) MaxAge() int {
 	if m.maxAge == nil {
-		m.maxAge = new(int)
-		*m.maxAge = extractMaxAge(m.rawHeader.Get("CACHE-CONTROL"), -1)
+		maxAge := extractMaxAge(m.rawHeader.Get("CACHE-CONTROL"), -1)
+		m.maxAge = &maxAge
```

Comment:
The original code is just to make it clear that we are allocating from the heap.
Please explain why we should use your version.

Reply:
There is not any specific improvement here over that I just find this way of doing it to be cleaner and easier to read. I find the setting of a field and then doing pointer dereference for setting the value to make the code harder to read. Also, I find that so many parts of the Go language can allocate on the heap that using `new` show that something is heap allocated doesn't make much sense. I usually avoid using that keyword entirely as I also find that it makes code harder to read.